me.hyliu.fluentreader.yml

```diff
@@ -0,0 +1,114 @@
+
+      # Run electron-builder, passing the architecture arguments to it.
+      # Note that the -- is important; without that, the argument will be passed to
+      # yarn instead of electron-builder.
+      - ". flatpak-node/electron-builder-arch-args.sh; npm run --prefix=fluent-reader --offline build; npm run --prefix=fluent-reader --offline package-linux-unpacked -- $ELECTRON_BUILDER_ARCH_ARGS"
+      # Copy the resulting, unpacked directory to /app.
+      # (A glob is used because the directory name may contain the current arch.)
+      - "cp -r fluent-reader/bin/linux/x64/linux*unpacked /app/fluent-reader"
+      # If you passed --electron-non-patented-ffmpeg, you could install it like this:
+      # - 'install -Dm 755 flatpak-node/libffmpeg.so -t /app/electron-webpack-quick-start'
+      # Install the wrapper script to start it.
+      - "install -Dm 755 start-fluent-reader.sh /app/bin/start-fluent-reader"
+    sources:
+      - type: git
+        url: https://github.com/xcffl/fluent-reader/
+        # Use the Electron 4 / Node 10 version.
+        commit: package-linux-unpacked
+        # Checkout into a subdirectory so we can copy the whole thing to /app.
+        dest: fluent-reader
+      # Add the flatpak-node-generator generated sources.
+      - generated-sources.json
+      # Our runner script.
+      - type: script
+        dest-filename: start-fluent-reader.sh
+        commands:
+          # Use zypak to call the electron binary to enable sandboxing and prevents no sandbox error
+          - "zypak-wrapper /app/fluent-reader/fluent-reader"
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 me.hyliu.fluentreader.appdata.xml /app/share/metainfo/me.hyliu.fluentreader.appdata.xml
+    sources:
+      - type: file
+        path: me.hyliu.fluentreader.appdata.xml
+  - name: desktop-entry
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 me.hyliu.fluentreader.desktop /app/share/applications/me.hyliu.fluentreader.desktop
+    sources:
+      - type: file
+        path: me.hyliu.fluentreader.desktop
+  - name: icons
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 me.hyliu.fluentreader-16.png /app/share/icons/hicolor/16x16/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-32.png /app/share/icons/hicolor/32x32/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-48.png /app/share/icons/hicolor/48x48/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-64.png /app/share/icons/hicolor/64x64/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-128.png /app/share/icons/hicolor/128x128/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-256.png /app/share/icons/hicolor/256x256/apps/me.hyliu.fluentreader.png
+      - install -Dm644 me.hyliu.fluentreader-512.png /app/share/icons/hicolor/512x512/apps/me.hyliu.fluentreader.png
```

Comment:
512x512 should be enough, Flatpak tooling will downscale it to 128x128 for displaying in the store.

Reply:
(Also, if there's a SVG file in git repo, please use it instead.)

Reply:
Will SVG in `scalable` be enough? Upstream can't use that because they build by `electron-builder` which only supports PNGs.

Reply:
Yeah, a file at `/app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg` should be enough.

Reply:
Done